src/components/standalone/interfaces_and_devices/ConfigureDeviceDrawer.vue

```diff
@@ -288,6 +293,51 @@ const devicesUsedByOhterBridgesOrBonds = computed(() => {
   return usedDevices
 })
 
+const primaryButtonLabel = computed(() => {
+  if (isConfiguringFromScratch.value) {
+    if (logicalIfaceType.value === 'bond') {
+      return t('standalone.interfaces_and_devices.create_bond')
+    } else if (props.device.proto === 'bonding') {
+      return t('standalone.interfaces_and_devices.configure_bond')
+    } else {
+      return t('standalone.interfaces_and_devices.configure_interface')
+    }
+  } else {
+    return t('standalone.interfaces_and_devices.reconfigure_interface')
+  }
+})
+
+const drawerTitle = computed(() => {
+  if (props.deviceType === 'physical') {
+    if (props.device.proto === 'bonding') {
+      const bondName = props.device.name.split('bond-')[1]
+      return t('standalone.interfaces_and_devices.configure_bond_name', {
+        name: bondName
+      })
+    } else {
+      return t('standalone.interfaces_and_devices.configure_interface_for_name', {
+        name: props.device.name
+      })
+    }
+  } else {
+    return t('standalone.interfaces_and_devices.create_logical_interface')
+  }
+})
+
+watch(
+  logicalIfaceType,
+  () => {
+    if (logicalIfaceType.value == 'bond') {
+      const firstRandom = Math.floor(Math.random() * 255)
+      const secondRandom = Math.floor(Math.random() * 255)
```

Comment:
Avoid to choose 0.

Reply:
[Following official documentation for .random() usage](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random#getting_a_random_integer_between_two_values), updating it now